### PR TITLE
[14.0][MIG] onchange_helper: Migration to 14.0

### PR DESCRIPTION
**Pull request by kevinkhao**
Trivial technical migration

### CODE REVIEW

**Review by sebastienbeau — APPROVED** [review]
LGTM code review

File: onchange_helper/README.rst [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+- Order in onchange_fields is very important as onchanges methods will be played in that order.
+- If you use memory object in `vals`, be award that onchange method in base model call `self.invalidate_cache()` that reset it.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-tools/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/server-tools/issues/new?body=module:%20onchange_helper%0Aversion:%2013.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (atchuthan) [inline onchange_helper/README.rst:84]: 13.0 to 14.0

File: onchange_helper/README.rst [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/server-tools <https://github.com/OCA/server-tools/tree/13.0/onchange_helper>`_ project on GitHub.
```
Comment (atchuthan) [inline onchange_helper/README.rst:118]: 13.0 to 14.0

File: onchange_helper/static/description/index.html [hunk 3]
```diff
@@ -0,0 +1,457 @@
...
+!! This file is generated by oca-gen-addon-readme !!
+!! changes will be overwritten.                   !!
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! -->
+<p><a class="reference external" href="https://odoo-community.org/page/development-status"><img alt="Beta" src="https://img.shields.io/badge/maturity-Beta-yellow.png" /></a> <a class="reference external" href="http://www.gnu.org/licenses/agpl-3.0-standalone.html"><img alt="License: AGPL-3" src="https://img.shields.io/badge/licence-AGPL--3-blue.png" /></a> <a class="reference external" href="https://github.com/OCA/server-tools/tree/13.0/onchange_helper"><img alt="OCA/server-tools" src="https://img.shields.io/badge/github-OCA%2Fserver--tools-lightgray.png?logo=github" /></a> <a class="reference external" href="https://translation.odoo-community.org/projects/server-tools-13-0/server-tools-13-0-onchange_helper"><img alt="Translate me on Weblate" src="https://img.shields.io/badge/weblate-Translate%20me-F47D42.png" /></a> <a class="reference external" href="https://runbot.odoo-community.org/runbot/149/13.0"><img alt="Try me on Runbot" src="https://img.shields.io/badge/runbot-Try%20me-875A7B.png" /></a></p>
```
Comment (atchuthan) [inline onchange_helper/static/description/index.html:370]: 13.0 to 14.0

**Review by atchuthan — APPROVED** [review]
minor changes needed in README.rst and index.html where version change from 13.0 to 14.0 is not done
Others look good :+1:

**Comment by kevinkhao** [comment]
@atchuthan I thought the oca bot did the oca-gen-addon-readme which refreshes the html and readme files ?

**Comment by rvalyi** [comment]
> @atchuthan I thought the oca bot did the oca-gen-addon-readme which refreshes the html and readme files ?

it does. @atchuthan we don't need to take care about the version in the generated README/description as it is generated properly by ocabot.

**Review by rvalyi — APPROVED** [review]

**Review by atchuthan — APPROVED** [review]
thanks for the info

**Review by PierrickBrun — APPROVED** [review]

**Pull request merged**